release/ray_release/byod/byod_increase_ray_max_limit_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
is this the recommended strategy for modifying the server env now?

Reply:
if it is before ray.init yes; otherwise using runtime_env is cheaper